ctru-rs/src/applets/error.rs

```diff
@@ -0,0 +1,146 @@
+//! Error applet
+//!
+//! This applet displays error text as a pop-up message on the lower screen.
+use crate::services::{apt::Apt, gfx::Gfx};
+
+use ctru_sys::errorConf;
+
+/// Configuration struct to set up the Error applet.
+#[doc(alias = "errorConf")]
+pub struct PopUp {
+    state: Box<errorConf>,
+}
+
+/// The kind of error applet to display.
+#[doc(alias = "errorType")]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+#[repr(u32)]
+pub enum Kind {
+    /// Error text is centered in the error applet window.
+    Center = ctru_sys::ERROR_TEXT,
+    /// Error text starts at the top of the error applet window.
+    Top = ctru_sys::ERROR_TEXT_WORD_WRAP,
```

Comment:
I might rename these after playing around with them some more. When I first experimented with them, it seemed like the word wrapping behavior was identical and only the text position was different. But I've since run into other examples that disprove that idea.

Reply:
But now the question is, what names are actually good here? `WordWrap` and `NoWordWrap`?

Reply:
I went with `WordWrap::Disabled` and `WordWrap::Enabled`